tutorials/tts/FastPitch_Finetuning.ipynb

```diff
@@ -634,15 +638,15 @@
                 "--config-name=hifigan.yaml \\\n",
                 "model.train_ds.dataloader_params.batch_size=32 \\\n",
                 "model.max_steps=1000 \\\n",
-                "model.optim.lr=0.0001 \\\n",
+                "model.optim.lr=0.00001 \\\n",
                 "~model.optim.sched \\\n",
                 "train_dataset=./hifigan_train_ft.json \\\n",
-                "validation_datasets=./hifigan_val_ft.json \\\n",
+                "validation_datasets=<your_hifigan_val_manifest> \\\n",
                 "exp_manager.exp_dir=hifigan_ft \\\n",
-                "+init_from_nemo_model=tts_hifigan.nemo \\\n",
+                "+init_from_pretrained_model=tts_hifigan \\\n",
                 "trainer.check_val_every_n_epoch=10 \\\n",
-                "model.train_ds=hifigan_train_ft.json \\\n",
-                "model.validation_ds=<your_hifigan_val_manifest>\n",
+                "model/train_ds=train_ds_finetune \\\n",
+                "model/validation_ds=val_ds_finetune\n",
```

Comment:
Why `/` instead of `.`?

Reply:
`model.train_ds` leads to errors and cannot be used to point to the train_ds config. The naming convention in hifigan.yaml file is using `/`
```
defaults:
  - model/generator: v1
  - model/train_ds: train_ds
  - model/validation_ds: val_ds
```

Reply:
Gotcha, thanks!